src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/AccessController.cs

```diff
@@ -176,32 +181,11 @@ string GetRedirectUrl()
         }
     }
 
-    private static void PopulateIdentityClaims(ClaimsPrincipal principal, ClaimsIdentity identity)
-    {
-        // Note: while ASP.NET Core Identity uses the legacy WS-Federation claims (exposed by the ClaimTypes class),
-        // OpenIddict uses the newer JWT claims defined by the OpenID Connect specification. To ensure the mandatory
-        // subject claim is correctly populated (and avoid an InvalidOperationException), it's manually added here.
-        if (!principal.HasClaim(static claim => claim.Type is Claims.Subject))
-        {
-            identity.AddClaim(new Claim(Claims.Subject, principal.GetUserIdentifier()));
-        }
-
-        if (!principal.HasClaim(static claim => claim.Type is Claims.Name))
-        {
-            identity.AddClaim(new Claim(Claims.Name, principal.GetUserName()));
-        }
-
-        if (!principal.HasClaim(static claim => claim.Type is Claims.Role))
-        {
-            foreach (var role in principal.GetRoles())
-            {
-                identity.AddClaim(new Claim(Claims.Role, role));
-            }
-        }
-    }
-
-    [ActionName(nameof(Authorize)), DisableCors]
-    [FormValueRequired("submit.Accept"), HttpPost]
+    [DisableCors]
+    [Route("connect/authorize/accept")]
```

Comment:
You can't do that, for two reasons:
  - The endpoint address is user-configurable, so it wouldn't make a lot of sense to have a dynamic path for the first action and static paths for the accept/deny actions.
  - The endpoint address MUST exactly match the URI configured in the server options. Otherwise, OpenIddict will throw an `InvalidOperationException` when trying to call `SignIn`.

Same thing for the logout endpoint.

Reply:
the `connect/authorize` option is still configurable. `connect/authorize/accept` is just so we can avoid the Ambiguity. 

Do you have a suggestion for fixing the issue?

Reply:
> the `connect/authorize` option is still configurable. `connect/authorize/accept` is just so we can avoid the Ambiguity.

As I mentioned in https://github.com/OrchardCMS/OrchardCore/issues/17472#issuecomment-2649326728, there should be no ambiguity as we're using `[FormValueRequired("submit.Accept")]` and `[FormValueRequired("submit.Deny")]` (which is the whole point of this thing: avoiding routing ambiguities by forcing specific form values to be present 😄).

If something is not working right, it's likely in https://github.com/OrchardCMS/OrchardCore/blob/3de1e7ff9e28a9b4afa1d25abe77bdd21e661ad2/src/OrchardCore/OrchardCore.Abstractions/Routing/FormValueRequiredMatcherPolicy.cs#L37-L64

Try to add a few BPs to see if it's correctly called?